src/komet/kdist/soroban-semantics/host/address.md

```diff
@@ -27,6 +27,19 @@ module HOST-ADDRESS
         <locals>
           0 |-> < i64 > _      // Address
         </locals>
+```
+
+## authorize_as_curr_contract
+
+```k
+    rule [hostfun-authorize-as-curr-contract]:
+        <instrs> hostCall ( "a" , "3" , [ i64  .ValTypes ] -> [ i64  .ValTypes ] )
```

Comment:
@bbyalcinkaya , let's make sure that we have this and the require_auth implementation in Komet's backlog.

Reply:
Done! Opened an issue for it: #75